replicate.Rmd

```diff
@@ -0,0 +1,1684 @@
+  summarise(mean.wtp = mean(wtp), CI.low = mean(wtp) -  (1.96 * Sem(wtp)), CI.high = mean(wtp) + (1.96 * Sem(wtp))) %>%
+  round(digits = 2)
+
+clean_1d_low_grocer <- clean_1d %>%
+  filter(split == 0, resort == 0) %>%
+  mutate(mean.wtp = mean(wtp), CI = (1.96 * Sem(wtp))) %>%
+  distinct(mean.wtp, CI)
+#for figure 1d
+```
+
+> The interaction between income and context was significant, F(1, 1984) = 4.82, p < 0.05, eta squared partial = 0.003.
+
+```{r echo = FALSE}
+summary(aov(wtp ~ resort * split, data = clean_1d))
+
+temp_eta <- etaSquared(aov(wtp ~ resort * split, data = clean_1d)) %>%
+  .[3,2] %>%
+  round(digit = 3)
+print(c("partial eta squared:", temp_eta), quote = FALSE)
+```
+
+> This interaction was marginally significant when income was treated continuously, beta = 0.50, t(1894) = 1.85, p < 0.07.
+
+```{r echo = FALSE}
+lm(scale(wtp) ~ scale(resort) + scale(logses) + scale(resort * logses), data = clean_1d) %>%
+  tidy() %>%
+  filter(term == "scale(resort * logses)") %>%
+  mutate(beta = round(estimate, digit = 2),
+         p = round(p.value, digit = 4)) %>%
+  select(term, beta, p)
+```
+
+```{r echo = FALSE}
+title = factor(c("Lower Income", "Higher Income"), levels = c("Lower Income", "Higher Income"))
+values = c(clean_1d_low_grocer$mean.wtp, clean_1d_high_grocer$mean.wtp, clean_1d_low_resort$mean.wtp, clean_1d_high_resort$mean.wtp)
+location = c("Grocer", "Grocer", "Resort", "Resort")
+CIs = c(clean_1d_low_grocer$CI, clean_1d_high_grocer$CI, clean_1d_low_resort$CI, clean_1d_high_resort$CI)
+fig1d.table <- data.frame(title, values, location, CIs)
+#figure 1d data frame
+
+figure1d <- fig1d.table %>%
+  ggplot(aes(x = title, y = values, fill = location)) + geom_bar(stat = "identity", position = position_dodge(), colour = "black") + ylab("WTP (dollars)") + xlab(NULL) + geom_errorbar(aes(ymin = values - CIs, ymax = values + CIs), width = 0.2, position = position_dodge(0.9)) + theme_bw() + scale_fill_grey(start = .3, end = .85) + ggtitle("Figure 1d - Amount participants were willing to pay (WTP)\nfor beer at the grocery store and resort in Study 1d\n(error bars represent 95% confidence interval)") + scale_y_continuous(breaks = seq(0, 8, 1)) + theme(panel.border = element_blank(), panel.grid.major = element_blank(), panel.grid.minor = element_blank(), axis.line = element_line(colour = "black"))
+#produces figure 1d
+
+print(figure1d)
+#displays figure 1d
+```
+
+> The beer-on-the-beach scenario provides a canonical example of the difficulty people have in translating utility into value.  People have a sense of how much they would enjoy the beer, but they have difficulty representing this enjoyment with a price.  However, under conditions of scarcity, people base their price not merely on anticipated enjoyment, but also on anticipated trade-offs, and those are more consistent guides for valuation.  Could scarcity also lead to more consistent baluation of a dollar itself? The next set of studies addressed this question.
+
```

Comment:
This seems like a paragraph better suited for a write up (to an actual article). Consider making code for different stages of analysis - from cleaning, to plots, to analysis for a project.